app/routers/organizations.py

```diff
@@ -43,8 +48,13 @@ async def create_organization(data: OrganizationCreate, organization_repo: Organ
             )
             response.raise_for_status()
             ffc_organization = response.json()
-            organization.organization_id = ffc_organization["id"]
-            return await organization_repo.update(organization.id, organization)
+            db_organization = await organization_repo.update(
+                db_organization.id,
+                {
+                    "organization_id": ffc_organization["id"],
+                },
+            )
+            return OrganizationRead.from_orm(db_organization)
     except httpx.HTTPStatusError as e:
```

Comment:
I would consider an alternative way to express the exceptions, using just one try/except block and inserting all the exception catching at the end.  What do you think?

Reply:
In this case we are returning two different errors with two different http statuses, so I think it's better to leave it as is to distinguish between errors received from the modifier from errors of the operations api itself.